docs/integrations/public-ip.md

```diff
@@ -0,0 +1,25 @@
+# Pipes' Public IP Addresses
+
+## **Overview**
+This document provides a list of public IPs used by Pipes for outbound requests. These IPs are intended for end-users to configure whitelisting, firewall rules, and integrations with external services.
```

Comment:
Are these for inbound, outbound or both?

Also, would just be `end users`?

Reply:
Yes, the ips are for both inbound and outbound requests, fixed it. Also changed `end-users` to `end users`